data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg

```diff
@@ -1373,23 +1461,12 @@ Rest-healing is an exception to the rule — if a unit doesn’t do anything for
 
         [message]
             speaker=unit
-            message= _ "Agh! Now who will teach you how to defeat these orcs!"
+            message= _ "Agh! Who will now guide you to the victory?"
```

Comment:
I would phrase this as 'Who will guide you to victory now?' At the very least should be just 'victory' instead of 'the victory'.